Apps/W1/HybridGP/app/src/pages/GPCompanyAddSettingsList.Page.al

```diff
@@ -18,137 +19,146 @@ page 4051 "GP Company Add. Settings List"
                     Caption = 'Rec. Master Only';
                     ToolTip = 'Specifies whether to migrate Receivables master data only.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Only Inventory Master"; Rec."Migrate Only Inventory Master")
                 {
                     Caption = 'Inventory Master Only';
                     ToolTip = 'Specifies whether to migrate Inventory master data only.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Inactive Customers"; Rec."Migrate Inactive Customers")
                 {
                     Caption = 'Inactive Customers';
                     ToolTip = 'Specifies whether to migrate inactive customers.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Inactive Vendors"; Rec."Migrate Inactive Vendors")
                 {
                     Caption = 'Inactive Vendors';
                     ToolTip = 'Specifies whether to migrate inactive vendors.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Inactive Checkbooks"; Rec."Migrate Inactive Checkbooks")
                 {
                     Caption = 'Inactive Checkbooks';
                     ToolTip = 'Specifies whether to migrate inactive checkbooks.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Inactive Items"; Rec."Migrate Inactive Items")
                 {
                     Caption = 'Inactive Items';
                     ToolTip = 'Specifies whether to migrate inactive items.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Discontinued Items"; Rec."Migrate Discontinued Items")
                 {
                     Caption = 'Discontinued Items';
                     ToolTip = 'Specifies whether to migrate discontinued items.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Customer Classes"; Rec."Migrate Customer Classes")
                 {
                     Caption = 'Customer Classes';
                     ToolTip = 'Specifies whether to migrate customer classes.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Vendor Classes"; Rec."Migrate Vendor Classes")
                 {
                     Caption = 'Vendor Classes';
                     ToolTip = 'Specifies whether to migrate vendor classes.';
-                    ApplicationArea = All;
                 }
                 field("Migrate Item Classes"; Rec."Migrate Item Classes")
                 {
                     Caption = 'Item Classes';
                     ToolTip = 'Specifies whether to migrate item classes.';
-                    ApplicationArea = All;
+                }
+                field("Oldest Hist. Year to Migrate"; Rec."Oldest Hist. Year to Migrate")
```

Comment:
Seems a little confusing when looking at "Oldest GL Year To Migrate" and this field. Going to make sense for someone else needing to work on this extension?

Reply:
How about "Initial Hist. Year"?

Reply:
same as other comment. leave it as is.